packages/backend-core/src/logging/pino/logger.ts

```diff
@@ -29,9 +35,9 @@ if (!env.DISABLE_PINO_LOGGER) {
 
   const destinations: pino.DestinationStream[] = []
 
-  if (env.isDev()) {
-    destinations.push(pinoPretty({ singleLine: true }))
-  }
+  destinations.push(
+    env.isDev() ? pinoPretty({ singleLine: true }) : process.stdout
```

Comment:
I think this works ok without this line? E.g. when I remove pino pretty, I still see logs in my terminal

Reply:
Actually it does not work for budibase services. It worked in account portal, where the logging returned to plaintext and is shown in the console. I'm not sure of why there is a difference

Reply:
ah - it's because of the `localFileDestination` - pino must default to console when no other streams are present